### ISSUE TITLE
Base js scripts files are not imported when an application is created from scratch.

### ISSUE THREAD

**Issue by leonardoavs** [jhipster/generator-jhipster#1159]
Currently when you generate a project from scratch, the angular and modules js files are not imported in the index.html file.

Chrome exception: Uncaught ReferenceError: angular is not defined
Environment: Microsoft windows.
This happens after the generation of a new app (basic application).
Jhipster version: 2.2.0

Files that needs to be included manually:

     <script src="bower_components/angular/angular.js"></script>
     <script src="bower_components/angular-dynamic-locale/tmhDynamicLocale.min.js" type="text/javascript"></script>
     <script src="bower_components/angular-local-storage/dist/angular-local-storage.js"></script>
     <script src="bower_components/angular-resource/angular-resource.js" type="text/javascript"></script>
     <script src="bower_components/angular-ui-router/release/angular-ui-router.js" type="text/javascript"></script>
     <script src="bower_components/angular-cookies/angular-cookies.js" type="text/javascript"></script>
     <script src="bower_components/angular-translate/angular-translate.js" type="text/javascript"></script>
     <script src="bower_components/angular-translate-storage-cookie/angular-translate-storage-cookie.js" type="text/javascript"></script>
     <script src="bower_components/angular-translate-loader-partial/angular-translate-loader-partial.js" type="text/javascript"></script>

This cause that the app only display the following text: This is your footer

**Comment by gmarziou** [comment]
This normally done automatically by the wiredep task of your grunt or gulp build. 
So probably somethign went wrong during your generation and you did not notice, it could be something failing during the npm install. Try to run `grunt build`

**Comment by jdubois** [comment]
Yes you probably had an error
Please try again and reopen this ticket if you still have an issue

**Comment by leonardoavs** [comment]
Hi,

This was my bad, I had an error with grunt, I want to know if there is a place in which we can discuss issues before put in the jhipster project?

For example, I have some issues with the openshift subgenerator of jhipster, and I want  to know if this issues will be resolved by someone in the future. 

The current jhipster implementation of openshift subgenerator, does not create an scalable application, and if you do not create an application scalable from the begining, it would not be scalable in the future. 

Moreover I saw, that when I tried to deploy to openshift the appication is restarting again and again.
